OmniBLE/PumpManager/MessageTransport.swift

```diff
@@ -262,23 +275,12 @@ class PodMessageTransport: MessageTransport {
 
         let response = try parseResponse(decrypted: decrypted)
 
-        /*if (!responseType.isInstance(response)) {
-            if (response is AlarmStatusResponse) {
-                throw PodAlarmException(response)
-            }
-            if (response is NakResponse) {
-                throw NakResponseException(response)
-            }
-            throw IllegalResponseException(responseType, response)
-        }
-         */
-
```

Comment:
I believe that there was some special stuff added added to Dash messaging for dealing with alerts at a higher level that this commented out code that is being deleted is referencing. Has this new Dash alert system been investigated at all?

Reply:
I'm not sure what you're referring to, or what this code was doing. If you have any clues, let me know.

Reply:
Looking at this again, this commented out code is taken from the Kotlin code here: https://github.com/nightscout/AndroidAPS/blob/master/omnipod-dash/src/main/java/info/nightscout/androidaps/plugins/pump/omnipod/dash/driver/comm/session/Session.kt. This used to be the place in AAPS where they were checking for an unexpected fault (alarm) response or a nak response from the pod. This checking has since apparently moved elsewhere for AAPS. For Loop this handling is handled at the next level up in PodCommSessions.send() as it was for Eros. So there is no problem removing this commented out code which isn't helpful or useful (and I agree that it would be good to do).

The special stuff for dealing with alerts that I had originally thought might be related was something completely different that I looked like was for dealing with the 8 pod alerts & the reminder beeps at a different level from what I remembered from some time back now. When I did some quick searches in the AAPS code, I couldn't find anything that looked like what I remember. This might have been something I saw in the decompiled PDM source which I can no longer find online. In any case, what I am remembering has nothing to do with this particular code being deleted.